packages/main/src/MonthPicker.js

```diff
@@ -131,6 +135,7 @@ class MonthPicker extends CalendarPart {
 				selected: isSelected,
 				ariaSelected: isSelected ? "true" : "false",
 				name: monthsNames[i],
+				nameInSecType: this.secondaryCalendarType && monthInSecType.text,
```

Comment:
Is there special reason **_getDisplayedSecondaryMonthText** to return object with single property (**.text**) instead of plain string?

Reply:
Done